src/main/resources/com/parasoft/findings/sonar/res/messages.properties

```diff
@@ -22,4 +22,6 @@ UnknownRuleName=Unknown rule
 UnsupportedFindingLanguage=Ignoring finding: {0} for unsupported language: {1}
 CoverageReportPathName=Parasoft Coverage Report Files
 CoverageReportPathDesc=Path (absolute or relative) to parasoft coverage xml report files.
-CoverageSubCategory=Parasoft Code Coverage
\ No newline at end of file
+CoverageSubCategory=Parasoft Code Coverage
+UploadCodeCoverageData=Uploading code coverage data: {0}
+InvalidReportFile=Invalid report file: {0}
```

Comment:
Since it's a warning message, I prefer to change it to something "gentle". 
For example:
Skipped invalid report file path: {0}